package.json

```diff
@@ -34,16 +32,20 @@
     "prepublishOnly": "npm run compile && npm run browser-compile && rimraf src",
     "postinstall": "node scripts/post-install.js"
   },
+  "peerDependencies": {
+    "graphql": "^0.10.0 || ^0.11.0"
+  },
   "devDependencies": {
     "@types/chai": "^4.0.0",
-    "@types/graphql": "^0.10.1",
+    "@types/graphql": "^0.11.2",
     "@types/lodash": "^4.14.68",
     "@types/mocha": "^2.2.41",
     "@types/node": "^8.0.8",
     "@types/sinon": "^2.3.0",
     "@types/ws": "^3.0.0",
     "chai": "^4.0.2",
-    "graphql": "^0.10.1",
+    "graphql": "^0.11.3",
+    "graphql-subscriptions": "^0.4.4",
```

Comment:
@DxCx update to latest :)

Reply:
yeah i was working at that since there was a typing still depended.. now it's pushed =)